web/app/configs/lang.json

```diff
@@ -801,16 +801,18 @@
         "emissionsImportedFrom": "{1} % av utsläppen i <img id=\"country-flag\"></img> <b>{2}</b> importeres från <img id=\"country-exchange-flag\"></img> <b>{3}</b>",
 
         "zoneShortName": {
+            "AD": "Andorra",
             "AL": "Albanien",
             "AM": "Armenien",
             "AT": "Österrike",
+            "AR": "Argentina",
             "AUS-NSW": "Australien (New South Wales)",
             "AUS-NT": "Australien (Northern Territory)",
             "AUS-QLD": "Australien (Queensland)",
-            "AUS-SA": "Australien (South Australien)",
+            "AUS-SA": "Australien (South Australia)",
```

Comment:
Shouldn't it be "Södra australien"?

Reply:
@corradio I have been unsure whether to translate this. When I look around the internet, some use the English name, and others a translated name.

To know how to do, I wrote to the Australian Embassy in Sweden and asked what they recommended. And they say i should use the English name.

Reply:
👍